### PR TITLE
Add powered by section to readme

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

Add a power by section based on https://amundsenworkspace.slack.com/archives/CGFBVT23V/p1561012353146600

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes. 
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)
- [ ] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @markgrover @ttannis @danwom @youngyjd

File: README.md [hunk 1]
```diff
@@ -29,30 +29,41 @@ or with an [Airflow DAG](https://github.com/lyft/amundsendatabuilder/blob/master
...
     ![](docs/img/data_preview.png)
 
 ## Get Involved in the Community
 
-Want help or want to help? 
+Want help or want to help?
 Use the button in our [header](https://github.com/lyft/amundsenfrontendlibrary#amundsen) to join our slack channel. Please join our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev) as well.
 
+## Powered By
+
```
Comment (markgrover) [inline README.md]: I'd just add a description line. These are some of the organizations that are using Amundsen today. If your organization uses Amundsen, please file a PR and update this list.

Comment (markgrover) [reply README.md]: Other than that looks great!

Comment (danwom) [reply README.md]: Doesn't Amundsen power the data discovery at X company? Seems like the relationship is going the wrong way with `Powered By`

Comment (feng-tao) [reply README.md]: Kafka used `Powered By`(https://kafka.apache.org/powered-by) while Airflow used `Who uses Apache Airflow?`(https://github.com/apache/airflow#who-uses-apache-airflow).

Which one do you guys prefer?

Comment (feng-tao) [reply README.md]: Updated:)

**Pull request merged**